src/strategy/organization-strategy.js

```diff
@@ -19,6 +19,24 @@ const getMultipleVersionsQuery = `
   }
 `
 
+const getMultipleVersionsQueryFirst100 = `
+  query getVersions($organization: String!, $names: [String!]!) {
+    organization(login: $organization) {
+      packages(first: 20, names: $names) {
+        nodes {
+          name
+          versions(first: 100, orderBy: {field: CREATED_AT, direction: DESC}) {
+            nodes {
+              id
+              version
+            }
+          }
+        }
+      }
+    }
+  }
+`
```

Comment:
It would be great if we could somehow avoid this duplication. Does it work if we do `last: 100` here but sort `ASC` instead of `DESC`? Then we could make the direction an argument and have only one query for the multiple versions case.

Reply:
ah I didn't think of that way of getting around this. I can look into it

Reply:
will need to pass the enums DESC and ASC